packages/website/pages/index.tsx

```diff
@@ -23,7 +19,7 @@ function Home({ componentsList }: HomeProps) {
           );
         })}
       </ul>
-    </>
+    </main>
```

Comment:
I think adding a `main` tag here will duplicate it unnecessarily since this component is rendered inside PageContent (line 158) which means that in the DOM this is how it will show:

```html
<main>
  <main>
    <h1>Welcome...</h1>
    <ul>....</ul>
  </main>
</main>
```

Reply:
`index.tsx` is not rendered inside `PageContent`. `PageContent` is used only for MDX based pages

Reply:
![Screenshot 2021-12-15 at 14 52 23](https://user-images.githubusercontent.com/6597467/146198851-d676f12a-4c3b-43ac-8191-5a6edd57327f.png)

I've just took this picture in vercel link

Reply:
There is also a main tag inside `Layout` (that's why I didn't want to abstract the Grid to another component)

the `Grid.Item` wrapping pages has `as="main"`

Reply:
And I forgot to mention, the advantage of removing this main here and keeping the one in Layout is that we make sure the title of each page is going to be inside of the main tag and screen readers and SEO like when the main content of a website has an `h1` inside of it

Reply:
got it, changed to article in `PostContent` and inde.tsx will be taken care of when working on Home screen